playbooks/force-to-enable-services.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2016 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+---
+- name: Install nodepool master node
+  become: yes
+  hosts: nodepool01
```

Comment:
Or maybe it's not good idea use nodepool01 here,  maybe here we can use nodepool-master?

Reply:
done

Reply:
And I think if you use nodepool01, then this playbook works for all deployment type.